communitybridge/2020/q2/selected_projects.md

```diff
@@ -62,3 +62,25 @@ Project maintainers and mentors, please submit the selected below (under the Sel
 - Issue:
   - [https://github.com/linkerd/linkerd2/issues/4325](https://github.com/linkerd/linkerd2/issues/4325)
 - Community Bridge project (URL): [https://people.communitybridge.org/project/65742dc0-7217-4c4a-a609-f5f0fcde5c0a](https://people.communitybridge.org/project/65742dc0-7217-4c4a-a609-f5f0fcde5c0a)
+
+### KubeEdge
+
+#### Support metrics-server in cloud
+
+- Description: Now we have integrated cadvisor in edge, and it also needs to integrate metrics-server in the cloud to collect monitoring data of edge nodes.
+- Recommended Skills: Golang, Kubernetes
+- Mentor(s): Kevin Wang (@kevin-wangzefeng)
+- Mentee: [Tiecheng Shen](https://github.com/Poor12)
+- Related Issues:
+  - https://github.com/kubeedge/kubeedge/issues/1561
+- Community Bridge project (URL): [https://people.communitybridge.org/project/1b931913-44a4-43a7-92ed-d7b2089060b1](https://people.communitybridge.org/project/1b931913-44a4-43a7-92ed-d7b2089060b1)
+
+#### Add certificate rotation for edge node
+
+- Description: At present, the edge nodes of KubeEdge only apply for certificates when they first started. And we need to automatically generate a new key and request a new certificate from the cloudcore periodically.
+- Recommended Skills: Golang, Kubernetes
+- Mentor(s): Kevin Wang (@kevin-wangzefeng), Fisher Xu (@fisherxu)
+- Mentee: [Jangel Xu](https://github.com/XJangel)
```

Comment:
Please replace the GitHub link with the Community Bridge one (https://people.communitybridge.org/mentee/...)

Reply:
Done.